FPR-0089/EVAL-0089.md

```diff
@@ -0,0 +1,57 @@
+
+# FPR Evaluation of FPR-0089
+
+Status - Partially Accepted - Changes required
+
+- [ ] **1. Has the FPR been submitted correctly?**
+   - [ ] A. Correctly titled?
+   - [ ] B. Used the full FPR template?
+   - [ ] C. Pull request correctly made to the BCF FPR repo?
+
+- [ ] **2. Have all sections of the template been completed?**
+   - [ ] A. Project Name
+   - [ ] B. Start Date
+   - [ ] C. Hashtag
+   - [ ] D. Name of BCF Chat room
+   - [ ] E. Stakeholders
+   - [ ] F. Project Summary
+   - [ ] G. Resources
+   - [ ] H. Budget
+   - [ ] I. Timeline
+   - [ ] J. Goals
+   - [ ] K. Other
+
+- [ ] **3. Is it well formatted for legibility?**
+
+- [ ] **4. Does the project have a non-profit purpose?**
+
+- [ ] **5. Has it been completed with enough detail for the scale of the project?**
+
+- [ ] **6. Is the team trustworthy enough for the funding requirement?**
+
+- [ ] **7. Are the objectives S.M.A.R.T?**
+   - [ ] A. Specific
+   - [ ] B. Measurable
+   - [ ] C. Agreed Upon
+   - [ ] D. Realistic
+   - [ ] E. Time-bound
+
+- [ ] **8. Are the stakeholders realistically providing volunteered time?**
+
+- [ ] **9. Is the cost vs impact satisfactory?**
+
+Cost = TODO
```

Comment:
The cost would be about 3000$ or less perhaps, we want to make conferences nationally, ie, be in the 24 cities of the country and that the days are free, give them graphic material to understand everything that bitcoin cash means.